airflow/cli/commands/internal_api_command.py

```diff
@@ -102,7 +102,7 @@ def internal_api(args):
             "--workers",
             str(num_workers),
             "--worker-class",
-            str(args.workerclass),
+            "uvicorn.workers.UvicornWorker",
```

Comment:
We should warn that provide `workerclass` has no affect anymore

Reply:
yes. That's in the TODOs.